modules/nf-core/freebayes/tests/main.nf.test

```diff
@@ -0,0 +1,179 @@
+nextflow_process {
+
+    name "Test Process FREEBAYES"
+    script "../main.nf"
+    process "FREEBAYES"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "freebayes"
+
+    test("sarscov2 - [ bam, bai ] - fasta - fai") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
+                    file(params.test_data['sarscov2']['illumina']['test_paired_end_sorted_bam'], checkIfExists: true),
+                    file(params.test_data['sarscov2']['illumina']['test_paired_end_sorted_bam_bai'], checkIfExists: true),
+                    [],
+                    [],
+                    []
+                ]
+                input[1] = [ [ id: 'test_fasta' ], file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true) ]
+                input[2] = [ [ id: 'test_fai' ],   file(params.test_data['sarscov2']['genome']['genome_fasta_fai'], checkIfExists: true) ]
+                input[3] = [ [], [] ]
+                input[4] = [ [], [] ]
+                input[5] = [ [], [] ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                // { assert snapshot(process.out).match() }, // Output VCF includes a timestamp, so snapshot not consistent past a day.
+                { assert snapshot(file(process.out.vcf.get(0).get(1)).name).match("test.vcf.gz") },
+                { assert path(process.out.vcf.get(0).get(1)).linesGzip.toString().contains('MT192765.1\t10214\t.\tATTTAC\tATTAC\t29.8242') },
+                { assert snapshot(file(process.out.versions.get(0)).name).match("versions.yml") },
```

Comment:
```suggestion
                { assert snapshot(process.out.versions).match() },
```

why capture only the name `versions.yml`......and not the hash of the file as above?

Reply:
Fixed versions assertions in all tests. Thanks!